features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/vitals/JankStatsActivityLifecycleListener.kt

```diff
@@ -24,17 +32,45 @@ import java.util.concurrent.TimeUnit
 internal class JankStatsActivityLifecycleListener(
     private val vitalObserver: VitalObserver,
     private val internalLogger: InternalLogger,
-    private val jankStatsProvider: JankStatsProvider = JankStatsProvider.DEFAULT
+    private val jankStatsProvider: JankStatsProvider = JankStatsProvider.DEFAULT,
+    private var screenRefreshRate: Double = 60.0
 ) : ActivityLifecycleCallbacks, JankStats.OnFrameListener {
 
     internal val activeWindowsListener = WeakHashMap<Window, JankStats>()
     internal val activeActivities = WeakHashMap<Window, MutableList<WeakReference<Activity>>>()
+    internal var display: Display? = null
```

Comment:
we should not keep the `Display` as a reference here as it can create memory leaks.

Reply:
if it is not bound to the particular UI-context and there is just a single instance for the whole app, it should be fine I guess.

Reply:
That was my assumption, I can replace with a weak reference if you prefer.

Reply:
if you are certain that it will be the main window `display` instance then it will be fine.